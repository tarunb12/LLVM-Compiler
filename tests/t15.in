int primeFactors(int n) {
    while (n % 2 == 0) {
        printf ("%d ", 2)
        n = n / 2
    }
    int i
    for (i = 3; i < n / 2; i += 2) {
        while (n % i == 0) {
            printf("%d ", i)
            n = n / i
        }
    }
    if (n > 2) printf("%d ", n)
    return 0
}

int main() {
    int n = 436
    primeFactors(n)
    return 0
}