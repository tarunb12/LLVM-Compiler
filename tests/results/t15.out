Error: File "tests/t15.in": Cannot redefine the parameter "n".